Multisig/UI/ClaimToken/Select Guardian/GuardianTableViewCell.swift

```diff
@@ -12,23 +12,28 @@ class GuardianTableViewCell: UITableViewCell {
     @IBOutlet private weak var addressInfoView: AddressInfoView!
     @IBOutlet private weak var descriptionLabel: UILabel!
     @IBOutlet private weak var containerView: UIView!
+    @IBOutlet private weak var checkMarkView: UIImageView!
+
+    var borderColor: UIColor = .border {
+        didSet {
+            setNeedsLayout()
+        }
+    }
 
     override func awakeFromNib() {
         super.awakeFromNib()
         containerView.layer.borderWidth = 2
         containerView.layer.cornerRadius = 10
-        containerView.layer.borderColor = UIColor.border.cgColor
         descriptionLabel.setStyle(.secondary)
     }
 
     override func layoutSubviews() {
         super.layoutSubviews()
         // changing here to react to dark/light color change
-        containerView.layer.borderColor = UIColor.border.cgColor
+        containerView.layer.borderColor = borderColor.cgColor
     }
 
-    func set(guardian: Guardian) {
-
+    func set(guardian: Guardian, selected: Bool = false) {
```

Comment:
Why don't we override setSelected method ?

Reply:
because this shows the contract selected method and not the "cell selection"